[2023-08-14T10:22:14.494+0200] {processor.py:157} INFO - Started process (PID=453363) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:22:14.495+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:22:14.496+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:22:14.496+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:22:14.501+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:22:14.500+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:22:14.501+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:22:14.501+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:22:14.502+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:22:14.510+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-08-14T10:23:02.075+0200] {processor.py:157} INFO - Started process (PID=453532) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:23:02.085+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:23:02.086+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:23:02.086+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:23:02.091+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:23:02.090+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:23:02.091+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:23:02.091+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:23:02.091+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:23:02.102+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T10:23:46.556+0200] {processor.py:157} INFO - Started process (PID=453787) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:23:46.565+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:23:46.566+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:23:46.565+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:23:46.571+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:23:46.571+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:23:46.572+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:23:46.572+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:23:46.572+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:23:46.586+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-08-14T10:24:28.810+0200] {processor.py:157} INFO - Started process (PID=453925) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:24:28.811+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:24:28.812+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:24:28.812+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:24:28.816+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:24:28.816+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:24:28.816+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:24:28.816+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:24:28.817+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:24:28.826+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-08-14T10:25:11.289+0200] {processor.py:157} INFO - Started process (PID=454071) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:25:11.298+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:25:11.300+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:25:11.299+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:25:11.303+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:25:11.303+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:25:11.304+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:25:11.303+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:25:11.304+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:25:11.317+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T10:26:01.122+0200] {processor.py:157} INFO - Started process (PID=454220) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:26:01.131+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:26:01.133+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:26:01.132+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:26:01.140+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:26:01.138+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:26:01.140+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:26:01.140+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:26:01.141+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:26:01.153+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-14T10:26:42.100+0200] {processor.py:157} INFO - Started process (PID=454486) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:26:42.109+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:26:42.110+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:26:42.110+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:26:42.116+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:26:42.115+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:26:42.117+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:26:42.116+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:26:42.117+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:26:42.132+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-14T10:27:22.141+0200] {processor.py:157} INFO - Started process (PID=454664) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:27:22.150+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:27:22.152+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:27:22.151+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:27:22.156+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:27:22.155+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:27:22.156+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:27:22.156+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:27:22.157+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:27:22.168+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T10:28:17.299+0200] {processor.py:157} INFO - Started process (PID=454936) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:28:17.309+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:28:17.310+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:28:17.309+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:28:17.315+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:28:17.314+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:28:17.316+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:28:17.316+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:28:17.317+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:28:17.328+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-08-14T10:28:58.998+0200] {processor.py:157} INFO - Started process (PID=455064) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:28:59.007+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:28:59.008+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:28:59.008+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:28:59.013+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:28:59.012+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:28:59.013+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:28:59.013+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:28:59.013+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:28:59.023+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-08-14T10:29:44.595+0200] {processor.py:157} INFO - Started process (PID=455232) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:29:44.604+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:29:44.605+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:29:44.605+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:29:44.610+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:29:44.609+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:29:44.610+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:29:44.610+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:29:44.611+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:29:44.623+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T10:30:35.724+0200] {processor.py:157} INFO - Started process (PID=455377) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:30:35.733+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:30:35.735+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:30:35.734+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:30:35.740+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:30:35.739+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:30:35.741+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:30:35.740+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:30:35.741+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:30:35.921+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.201 seconds
[2023-08-14T10:31:15.607+0200] {processor.py:157} INFO - Started process (PID=455493) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:31:15.616+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:31:15.617+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:31:15.617+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:31:15.623+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:31:15.621+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:31:15.623+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:31:15.623+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:31:15.623+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:31:15.634+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T10:32:05.038+0200] {processor.py:157} INFO - Started process (PID=455647) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:32:05.047+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:32:05.048+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:32:05.048+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:32:05.054+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:32:05.053+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:32:05.054+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:32:05.054+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:32:05.055+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:32:05.065+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T10:32:55.368+0200] {processor.py:157} INFO - Started process (PID=455928) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:32:55.378+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:32:55.379+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:32:55.379+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:32:55.384+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:32:55.383+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:32:55.385+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:32:55.384+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:32:55.385+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:32:55.396+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T10:33:47.557+0200] {processor.py:157} INFO - Started process (PID=456058) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:33:47.566+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:33:47.568+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:33:47.567+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:33:47.573+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:33:47.572+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:33:47.574+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:33:47.573+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:33:47.574+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:33:47.583+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T10:34:42.745+0200] {processor.py:157} INFO - Started process (PID=456185) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:34:42.746+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:34:42.747+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:34:42.747+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:34:42.753+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:34:42.752+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:34:42.754+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:34:42.754+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:34:42.754+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:34:42.764+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-08-14T10:35:35.061+0200] {processor.py:157} INFO - Started process (PID=456350) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:35:35.070+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:35:35.071+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:35:35.071+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:35:35.077+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:35:35.076+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:35:35.077+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:35:35.077+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:35:35.077+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:35:35.089+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T10:36:25.971+0200] {processor.py:157} INFO - Started process (PID=456494) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:36:25.973+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:36:25.974+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:36:25.974+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:36:25.980+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:36:25.979+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:36:25.981+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:36:25.981+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:36:25.981+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:36:25.991+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-08-14T10:37:14.233+0200] {processor.py:157} INFO - Started process (PID=456609) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:37:14.242+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:37:14.243+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:37:14.243+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:37:14.248+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:37:14.247+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:37:14.248+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:37:14.248+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:37:14.249+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:37:14.260+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-08-14T10:38:04.154+0200] {processor.py:157} INFO - Started process (PID=456745) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:38:04.163+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:38:04.164+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:38:04.164+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:38:04.169+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:38:04.169+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:38:04.170+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:38:04.170+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:38:04.170+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:38:04.178+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-08-14T10:38:55.296+0200] {processor.py:157} INFO - Started process (PID=456862) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:38:55.305+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:38:55.306+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:38:55.305+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:38:55.311+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:38:55.310+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:38:55.311+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:38:55.311+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:38:55.312+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:38:55.327+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-08-14T10:39:46.427+0200] {processor.py:157} INFO - Started process (PID=457122) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:39:46.429+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:39:46.430+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:39:46.429+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:39:46.434+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:39:46.433+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:39:46.434+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:39:46.434+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:39:46.434+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:39:46.445+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-08-14T10:41:28.177+0200] {processor.py:157} INFO - Started process (PID=457545) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:41:28.178+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:41:28.179+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:41:28.179+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:41:28.184+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:41:28.183+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:41:28.185+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:41:28.185+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:41:28.185+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:41:28.194+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-08-14T10:42:13.784+0200] {processor.py:157} INFO - Started process (PID=457707) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:42:13.785+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:42:13.786+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:42:13.786+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:42:13.788+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:42:13.788+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:42:13.788+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:42:13.788+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:42:13.788+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:42:13.795+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2023-08-14T10:44:20.063+0200] {processor.py:157} INFO - Started process (PID=458207) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:44:20.072+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:44:20.073+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:44:20.073+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:44:20.079+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:44:20.078+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:44:20.079+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:44:20.079+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:44:20.080+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:44:20.090+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T10:45:05.580+0200] {processor.py:157} INFO - Started process (PID=458431) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:45:05.588+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:45:05.590+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:45:05.589+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:45:05.595+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:45:05.594+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:45:05.595+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:45:05.595+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:45:05.596+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:45:05.606+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-08-14T10:45:51.765+0200] {processor.py:157} INFO - Started process (PID=458641) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:45:51.767+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:45:51.768+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:45:51.768+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:45:51.773+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:45:51.772+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:45:51.774+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:45:51.773+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:45:51.774+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:45:51.785+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-08-14T10:46:39.517+0200] {processor.py:157} INFO - Started process (PID=458854) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:46:39.526+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:46:39.527+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:46:39.527+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:46:39.532+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:46:39.531+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:46:39.532+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:46:39.532+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:46:39.532+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:46:39.545+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-08-14T10:47:28.760+0200] {processor.py:157} INFO - Started process (PID=459009) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:47:28.761+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:47:28.763+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:47:28.762+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:47:28.768+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:47:28.767+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:47:28.768+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:47:28.768+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:47:28.769+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:47:28.779+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-08-14T10:48:23.997+0200] {processor.py:157} INFO - Started process (PID=459214) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:48:24.006+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:48:24.008+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:48:24.007+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:48:24.013+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:48:24.012+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:48:24.013+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:48:24.013+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:48:24.014+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:48:24.025+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-08-14T10:49:17.582+0200] {processor.py:157} INFO - Started process (PID=459390) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:49:17.591+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:49:17.593+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:49:17.592+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:49:17.598+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:49:17.597+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:49:17.598+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:49:17.598+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:49:17.599+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:49:17.611+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-08-14T10:50:07.190+0200] {processor.py:157} INFO - Started process (PID=459536) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:50:07.199+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:50:07.201+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:50:07.200+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:50:07.206+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:50:07.205+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:50:07.206+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:50:07.206+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:50:07.207+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:50:07.219+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T10:50:58.909+0200] {processor.py:157} INFO - Started process (PID=459764) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:50:58.911+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:50:58.911+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:50:58.911+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:50:58.913+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:50:58.913+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:50:58.914+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:50:58.913+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:50:58.914+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:50:58.921+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2023-08-14T10:51:48.795+0200] {processor.py:157} INFO - Started process (PID=459889) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:51:48.804+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:51:48.805+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:51:48.805+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:51:48.811+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:51:48.810+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:51:48.811+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:51:48.811+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:51:48.812+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:51:48.822+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T10:52:38.005+0200] {processor.py:157} INFO - Started process (PID=460005) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:52:38.006+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:52:38.007+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:52:38.006+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:52:38.011+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:52:38.010+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:52:38.011+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:52:38.011+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:52:38.012+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:52:38.019+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-08-14T10:53:28.547+0200] {processor.py:157} INFO - Started process (PID=460215) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:53:28.556+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:53:28.558+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:53:28.557+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:53:28.562+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:53:28.561+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:53:28.562+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:53:28.562+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:53:28.562+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:53:28.579+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2023-08-14T10:54:21.366+0200] {processor.py:157} INFO - Started process (PID=460401) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:54:21.375+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:54:21.376+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:54:21.376+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:54:21.381+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:54:21.380+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:54:21.382+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:54:21.381+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:54:21.382+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:54:21.395+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2023-08-14T10:55:12.227+0200] {processor.py:157} INFO - Started process (PID=460577) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:55:12.236+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:55:12.238+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:55:12.237+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:55:12.242+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:55:12.241+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:55:12.243+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:55:12.243+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:55:12.243+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:55:12.254+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T10:56:02.105+0200] {processor.py:157} INFO - Started process (PID=460710) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:56:02.114+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:56:02.115+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:56:02.115+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:56:02.121+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:56:02.120+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:56:02.121+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:56:02.121+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:56:02.121+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:56:02.136+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-14T10:56:50.080+0200] {processor.py:157} INFO - Started process (PID=460927) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:56:50.089+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:56:50.090+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:56:50.089+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:56:50.094+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:56:50.093+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:56:50.094+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:56:50.094+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:56:50.094+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:56:50.106+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T10:57:39.686+0200] {processor.py:157} INFO - Started process (PID=461058) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:57:39.695+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:57:39.696+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:57:39.696+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:57:39.700+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:57:39.700+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:57:39.701+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:57:39.701+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:57:39.701+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:57:39.712+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-08-14T10:58:23.182+0200] {processor.py:157} INFO - Started process (PID=461288) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:58:23.191+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:58:23.192+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:58:23.192+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:58:23.198+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:58:23.197+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:58:23.198+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:58:23.198+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:58:23.198+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:58:23.208+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T10:59:12.885+0200] {processor.py:157} INFO - Started process (PID=461433) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:59:12.887+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T10:59:12.887+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:59:12.887+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:59:12.890+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:59:12.890+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T10:59:12.890+0200] {logging_mixin.py:150} INFO - [2023-08-14T10:59:12.890+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T10:59:12.891+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T10:59:12.899+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2023-08-14T11:00:01.209+0200] {processor.py:157} INFO - Started process (PID=461619) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:00:01.219+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:00:01.220+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:00:01.220+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:00:01.228+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:00:01.227+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:00:01.228+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:00:01.228+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:00:01.228+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:00:01.236+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T11:00:50.258+0200] {processor.py:157} INFO - Started process (PID=461737) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:00:50.267+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:00:50.268+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:00:50.267+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:00:50.272+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:00:50.272+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:00:50.273+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:00:50.273+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:00:50.273+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:00:50.285+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T11:01:45.707+0200] {processor.py:157} INFO - Started process (PID=461958) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:01:45.709+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:01:45.710+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:01:45.710+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:01:45.713+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:01:45.713+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:01:45.713+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:01:45.713+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:01:45.714+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:01:45.722+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-08-14T11:02:39.179+0200] {processor.py:157} INFO - Started process (PID=462124) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:02:39.189+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:02:39.190+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:02:39.189+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:02:39.196+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:02:39.195+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:02:39.196+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:02:39.196+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:02:39.196+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:02:39.211+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-08-14T11:03:29.904+0200] {processor.py:157} INFO - Started process (PID=462269) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:03:29.913+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:03:29.914+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:03:29.914+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:03:29.921+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:03:29.919+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:03:29.921+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:03:29.921+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:03:29.921+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:03:29.934+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-14T11:04:20.213+0200] {processor.py:157} INFO - Started process (PID=462394) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:04:20.222+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:04:20.223+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:04:20.223+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:04:20.228+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:04:20.227+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:04:20.229+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:04:20.228+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:04:20.229+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:04:20.244+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-14T11:05:13.128+0200] {processor.py:157} INFO - Started process (PID=462536) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:05:13.137+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:05:13.138+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:05:13.138+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:05:13.144+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:05:13.143+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:05:13.144+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:05:13.144+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:05:13.145+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:05:13.154+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T11:06:06.741+0200] {processor.py:157} INFO - Started process (PID=462686) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:06:06.750+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:06:06.751+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:06:06.750+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:06:06.756+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:06:06.755+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:06:06.756+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:06:06.756+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:06:06.756+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:06:06.764+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2023-08-14T11:06:52.955+0200] {processor.py:157} INFO - Started process (PID=462886) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:06:52.964+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:06:52.966+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:06:52.965+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:06:52.971+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:06:52.970+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:06:52.971+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:06:52.971+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:06:52.972+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:06:52.983+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T11:07:48.388+0200] {processor.py:157} INFO - Started process (PID=463050) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:07:48.397+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:07:48.398+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:07:48.398+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:07:48.403+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:07:48.402+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:07:48.403+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:07:48.403+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:07:48.404+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:07:48.415+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T11:08:36.548+0200] {processor.py:157} INFO - Started process (PID=463205) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:08:36.557+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:08:36.559+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:08:36.558+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:08:36.564+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:08:36.563+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:08:36.565+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:08:36.565+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:08:36.565+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:08:36.577+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T11:10:45.855+0200] {processor.py:157} INFO - Started process (PID=463635) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:10:45.864+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:10:45.865+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:10:45.865+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:10:45.873+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:10:45.872+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:10:45.874+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:10:45.874+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:10:45.874+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:10:45.885+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-14T11:11:49.378+0200] {processor.py:157} INFO - Started process (PID=463992) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:11:49.380+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:11:49.381+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:11:49.380+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:11:49.386+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:11:49.385+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:11:49.387+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:11:49.387+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:11:49.387+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:11:49.395+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2023-08-14T11:12:34.985+0200] {processor.py:157} INFO - Started process (PID=464154) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:12:34.994+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:12:34.996+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:12:34.995+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:12:34.999+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:12:34.998+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:12:34.999+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:12:34.999+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:12:34.999+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:12:35.011+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T11:13:37.380+0200] {processor.py:157} INFO - Started process (PID=464512) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:13:37.381+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:13:37.382+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:13:37.382+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:13:37.385+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:13:37.385+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:13:37.385+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:13:37.385+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:13:37.385+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:13:37.401+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-08-14T11:15:15.343+0200] {processor.py:157} INFO - Started process (PID=464834) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:15:15.352+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:15:15.353+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:15:15.352+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:15:15.358+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:15:15.357+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:15:15.359+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:15:15.359+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:15:15.359+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:15:15.368+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2023-08-14T11:16:01.896+0200] {processor.py:157} INFO - Started process (PID=465003) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:16:01.906+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:16:01.907+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:16:01.907+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:16:01.913+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:16:01.912+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:16:01.913+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:16:01.913+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:16:01.914+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:16:01.923+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T11:16:50.586+0200] {processor.py:157} INFO - Started process (PID=465131) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:16:50.595+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:16:50.596+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:16:50.595+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:16:50.601+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:16:50.600+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:16:50.601+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:16:50.601+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:16:50.601+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:16:50.614+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2023-08-14T11:17:43.393+0200] {processor.py:157} INFO - Started process (PID=465269) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:17:43.402+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:17:43.403+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:17:43.402+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:17:43.409+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:17:43.408+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:17:43.409+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:17:43.409+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:17:43.409+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:17:43.419+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T11:18:36.212+0200] {processor.py:157} INFO - Started process (PID=465507) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:18:36.221+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:18:36.222+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:18:36.222+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:18:36.228+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:18:36.227+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:18:36.228+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:18:36.228+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:18:36.228+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:18:36.238+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.030 seconds
[2023-08-14T11:20:16.267+0200] {processor.py:157} INFO - Started process (PID=465841) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:20:16.276+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:20:16.277+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:20:16.277+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:20:16.283+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:20:16.282+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:20:16.283+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:20:16.283+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:20:16.283+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:20:16.292+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2023-08-14T11:21:10.798+0200] {processor.py:157} INFO - Started process (PID=466063) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:21:10.799+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:21:10.800+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:21:10.800+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:21:10.805+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:21:10.804+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:21:10.805+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:21:10.805+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:21:10.805+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:21:10.815+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-08-14T11:21:58.908+0200] {processor.py:157} INFO - Started process (PID=466221) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:21:58.917+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:21:58.918+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:21:58.918+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:21:58.924+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:21:58.923+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:21:58.924+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:21:58.924+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:21:58.925+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:21:58.939+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2023-08-14T11:22:52.917+0200] {processor.py:157} INFO - Started process (PID=466431) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:22:52.926+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:22:52.927+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:22:52.927+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:22:52.932+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:22:52.931+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:22:52.933+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:22:52.932+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:22:52.933+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:22:52.945+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-08-14T11:23:41.217+0200] {processor.py:157} INFO - Started process (PID=467026) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:23:41.226+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:23:41.227+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:23:41.227+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:23:41.231+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:23:41.230+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:23:41.231+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:23:41.231+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:23:41.231+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:23:41.243+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-08-14T11:24:30.935+0200] {processor.py:157} INFO - Started process (PID=467217) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:24:30.945+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:24:30.946+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:24:30.946+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:24:30.954+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:24:30.953+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:24:30.954+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:24:30.954+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:24:30.955+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:24:30.967+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2023-08-14T11:25:19.685+0200] {processor.py:157} INFO - Started process (PID=467635) to work on /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:25:19.694+0200] {processor.py:826} INFO - Processing file /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-14T11:25:19.695+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:25:19.695+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:25:19.700+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:25:19.699+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-14T11:25:19.700+0200] {logging_mixin.py:150} INFO - [2023-08-14T11:25:19.700+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-14T11:25:19.701+0200] {processor.py:838} WARNING - No viable dags retrieved from /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-14T11:25:19.713+0200] {processor.py:179} INFO - Processing /home/akhilesh/py39/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
